### DISCUSSION TITLE
Configuring new robot with 7+ DOF and/or with prismatic joints

### DISCUSSION THREAD

**Discussion by jsupratman13** [NVlabs/curobo#36] [Software Q&A] [answered]
Hi, I have a custom robot with 8DOF, with one translating joint. I was wondering if cuRobo supports more than 7DOF and if it also supports prismatic joints.

**Comment by balakumar-s** [comment, marked as answer]
Yes, it does support up to 128 joints. It also supports prismatic joints.
